### PR TITLE
Dashboard base functionality

### PR DESCRIPTION
**Pull request by vidarshan**
# Dashboard base functionality

- Display transaction list on dashboard home
- Display an information card for the community

### CODE REVIEW

File: src/containers/CommunityHome/index.tsx [hunk 1]
```diff
@@ -11,32 +10,304 @@ import {
...
+                    return (
+                      <TableRow key={transaction.hash}>
+                        <TableCell>
+                          {shortenAddress(transaction?.hash)}
+                        </TableCell>
+                        <TableCell>
+                          {shortenAddress(transaction?.tx_hash)}
+                        </TableCell>
+                        <TableCell>
+                          {moment(transaction?.created_at).format(
```
Comment (kevtechi) [inline src/containers/CommunityHome/index.tsx]: There's no need to use moment anymore for date formatting. It's built into JS now.

```
const date = new Date();
const options = {
  year: 'numeric',
  month: 'long',
  day: 'numeric',
  weekday: 'long',
  hour: '2-digit',
  minute: '2-digit',
  second: '2-digit',
  timeZoneName: 'short'
};

const locale = navigator.language;

const formattedDateTime = new Intl.DateTimeFormat(locale, options).format(date);
console.log(formattedDateTime);
```

Comment (kevtechi) [reply src/containers/CommunityHome/index.tsx]: Maybe you can put this in a helper function and call it where it's needed?

Comment (kevtechi) [reply src/containers/CommunityHome/index.tsx]: Then you can remove the `moment` dependency.

**Review by kevtechi — CHANGES_REQUESTED** [review]

File: src/containers/CommunityHome/index.tsx (hunk 1 above)
Comment (vidarshan) [reply src/containers/CommunityHome/index.tsx]: sure @kevtechi

File: src/containers/CommunityHome/index.tsx [hunk 2]
```diff
@@ -11,32 +10,304 @@ import {
...
+                </IconButton>
+              </Box>
+              <Box>
+                <IconButton
+                  className="m-2 cursor-pointer"
+                  variant="soft"
+                  color="green"
+                >
+                  <Download />
+                </IconButton>
```
Comment (kevtechi) [inline src/containers/CommunityHome/index.tsx]: Let's not add this button yet. When we implement downloading, we can add it.

**Review by kevtechi — CHANGES_REQUESTED** [review]

File: src/containers/CommunityHome/index.tsx [hunk 3]
```diff
@@ -11,32 +10,304 @@ import {
...
+                        <TableCell>
+                          {moment(transaction?.created_at).format(
+                            "DD-MMM-YYYY"
+                          )}
+                        </TableCell>
+                        <TableCell>
+                          {shortenAddress(transaction?.from)}
+                        </TableCell>
+                        <TableCell>{shortenAddress(transaction?.to)}</TableCell>
+                        <TableCell>{transaction?.value}</TableCell>
```
Comment (kevtechi) [inline src/containers/CommunityHome/index.tsx]: for this, the amount needs to be formatted for decimals.

```
import { formatUnits } from "ethers";

formatUnits(`${transaction?.value ?? 0}`, community.token.decimals)
```

**Review by kevtechi — CHANGES_REQUESTED** [review]

File: src/containers/CommunityHome/index.tsx [hunk 4]
```diff
@@ -11,32 +10,304 @@ import {
...
+              <TableBody>
+                {transfers &&
+                  transfers?.map((transaction: Transfer) => {
+                    return (
+                      <TableRow key={transaction.hash}>
+                        <TableCell>
+                          {shortenAddress(transaction?.hash)}
+                        </TableCell>
+                        <TableCell>
+                          {shortenAddress(transaction?.tx_hash)}
```
Comment (kevtechi) [inline src/containers/CommunityHome/index.tsx]: Could you make this clickable, like a link?

Url format: `${community.scan.url}/tx/${transaction?.tx_hash}`

Comment (kevtechi) [reply src/containers/CommunityHome/index.tsx]: Look at the table on this page: https://celoscan.io/token/0x3c960e72bbbd837293e75080e1d0fee6a4640357

**Review by kevtechi — CHANGES_REQUESTED** [review]

File: src/containers/CommunityHome/index.tsx (hunk 4 above)
Comment (vidarshan) [reply src/containers/CommunityHome/index.tsx]: done

**Comment by kevtechi** [comment]
Looks good. It can be merged.

**Pull request merged**